integration_tests/ci/sample.profiles.yml

```diff
@@ -12,13 +12,13 @@ integration_tests:
       pass: "{{ env_var('CI_REDSHIFT_DBT_PASS') }}"
       dbname: "{{ env_var('CI_REDSHIFT_DBT_DBNAME') }}"
       port: 5439
-      schema: intercom_integration_tests
+      schema: intercom_integration_tests_2
```

Comment:
Did you try dropping the old schema via drop schema cascade instead of creating new ones? At some discussion we were trying to limit having to do changes like this.